xml/cap_depl_eks.xml

```diff
@@ -370,151 +288,347 @@ Custom TCP Rule    TCP          4443            0.0.0.0/0       CAP WSS
+          <thead>
+              <row>
+                  <entry>&kube; service descriptions</entry>
+                  <entry>&kube; service names</entry>
+              </row>
+          </thead>
+          <tbody>
+              <row>
+                  <entry>User Account and Authentication (UAA)</entry>
+                  <entry>uaa-uaa-public</entry>
+               </row>
+               <row>
+                  <entry>Cloud Foundry (CF) TCP routing service</entry>
+                  <entry>tcp-router-tcp-router-public</entry>
+               </row>
+              <row>
+                  <entry>CF application SSH access</entry>
+                  <entry>diego-ssh-ssh-proxy-public</entry>
+               </row>
+               <row>
+                  <entry>CF router</entry>
+                  <entry>router-gorouter-public</entry>
+               </row>
+           </tbody>
+       </tgroup>
+   </informaltable>
+   <para>
+       <literal>uaa-uaa-public</literal> is in the <literal>uaa</literal> 
+       namespace, and the rest are in the <literal>scf</literal> namespace.
+   </para>
+         <para>
+         After deployment your domain must be changed to the load 
+         balancer IP address. The following table lists the required domain and 
+         sub-domains, using <literal>example.com</literal> as the example domain:
+     </para>
+  <informaltable>
+      <tgroup cols="2">
+          <thead>
+              <row>
+                  <entry>Domains</entry>
+                  <entry>Services</entry>
+              </row>
+          </thead>
+          <tbody>
+              <row>
+                  <entry>example.com</entry>
+                  <entry>scf/router-public</entry>
+               </row>
+               <row>
+                  <entry>*.example.com</entry>
+                  <entry>scf/router-public</entry>
+               </row>
+               <row>
+                  <entry>uaa.example.com</entry>
+                  <entry>uaa/uaa-public</entry>
+               </row>
+               <row>
+                  <entry>*.uaa.example.com</entry>
+                  <entry>uaa/uaa-public</entry>
+               </row>                
+              <row>
+                  <entry>tcp.example.com</entry>
+                  <entry>scf/tcp-router-public</entry>
+               </row>
+               <row>
+                  <entry>ssh.example.com</entry>
+                  <entry>scf/diego-access-public</entry>
+               </row>
+           </tbody>
+       </tgroup>
+   </informaltable>
+</sect1>
+ 
+<sect1 xml:id="sec.cap.eks-configuration">
+    <title>Deployment Configuration</title>
   <para>
    Use this example <filename>scf-config-values.yaml</filename> as a template
    for your configuration.
   </para>
 
 <screen>env:
-  # Domain for SCF. DNS for *.DOMAIN must point to a kube node's (not master)
-  # external IP address.
   DOMAIN: <replaceable>public_ip_of_node-VM.example.com</replaceable>
```

Comment:
Shouldn't this just be `example.com` (since that's what the [DNS configuration](https://github.com/SUSE/doc-cap/pull/344/files#diff-bbc27e7263868765ef98bb8c6220ef8aR365) says)?

Reply:
:heavy_check_mark: